pants

```diff
@@ -76,7 +76,7 @@ PANTS_SRCPATH="$(echo ${PANTS_SRCPATH[@]} | tr ' ' :)"
 function exec_pants_bare() {
   # Redirect activation and native bootstrap to ensure that they don't interfere with stdout.
   activate_pants_venv 1>&2
-  bootstrap_native_code 1>&2
+  #bootstrap_native_code 1>&2
```

Comment:
Ah please do not commit this. That's why CI was failing. Also if you merge with the latest master, this workaround should not be needed anymore.